app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -107,86 +131,37 @@ Please note that you will not be able to use eligibility for the international r
-
-#### Scholarships for physics, French, German and Spanish
-
-To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process.
-
-If you’re successful, your teacher training provider will pay the scholarship money directly into your bank account over a period of 10 months.
-
-You may get a:
-
-* physics scholarship of £30,000
-* French, German or Spanish scholarship of £27,000
-
-To apply for a scholarship, you’ll need a place on a course training to teach French, German, Spanish or physics with an accredited teacher training provider in England, in the academic year 2024 to 2025.
-
-You’ll also need at least one of the following:
-
-* a bachelor’s degree class 2:1 (honours) or higher
-* a master’s degree
-* a PhD
-
-In exceptional circumstances, you may be offered a scholarship with a bachelor’s degree class 2:2, if you have significant relevant experience.
-You can check if you’re eligible for a scholarship and apply by visiting:
-
-* the [Institute of Physics](https://www.iop.org/about/support-grants/iop-teacher-training-scholarships#gref) (for a physics scholarship) 
-* the [British Council](https://www.britishcouncil.org/education/he-science/opportunities/ltts) (for a French, German or Spanish scholarship)
-
-[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in languages and physics.
-
-
-
-## Student loans
-
-Most non-UK candidates will not be eligible for student loans to cover fees and living expenses. If you're an Irish citizen or have some type of settled status in the UK – for example, through the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain), some financial help is available.  
-
-[Check your eligibility for student loans](https://www.gov.uk/student-finance/who-qualifies).
-
-If you're eligible for student loans you may also be eligible for a [teacher training bursary](/funding-and-support/scholarships-and-bursaries). 
 
 ## How to apply for student loans, bursaries and scholarships 
 
 To start your application for student loans, you need to have chosen your teacher training course, but you do not need a confirmed place. 
 
+[Check your eligibility](https://www.gov.uk/student-finance/who-qualifies).
+
 [Apply to Student Finance England](https://www.gov.uk/student-finance) for:
 
 * tuition fee loans
-* maintenance loans
+* maintenance loans to help with living costs
 * extra help if you have a disability, children or an adult dependant
 
+If you are eligible for student loans, you will also usually be eligible: 
+
+* for a home fee rate 
+
+* for bursaries in the subjects offered to UK citizens 
+
+* to apply for scholarships 
+
 Talk to your training provider about:
 
```

Comment:
@laurahermionetennant92 does this last talk to your training provider section need to be in bullet points? Could ti just be a sentence at the end?

Reply:
yes that's better